api/Trace/SpanContext.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Trace;
+
+interface SpanContext
+{
+    function getTraceId(): string;
+    function getSpanId(): string;
+    function getTraceFlags(): int;
+    function getTracestate(): array;
+    function IsValidContext(): bool;
```

Comment:
since we are in SpanContext, do we need Context suffix for IsValid and IsRemote?

Reply:
Yes! I hope that SDKs will implement SpanContext on the Span not use a separate object for the context unless specifically asked for. Again, for memory reasons. Originally I made this a requirement, but relaxed it. What do you think?